Azure-Samples/AzureMapsCodeSamples
# Vnext

## Purpose

GeoJSON routing sample was broken due to broken script URLs in the sample as a result of the recent change in solution architecture.

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

1. Go to the Spatial Analysis -> GeoJSON Routing sample and see that it loads without any 404 errors in console for script files.
